src/page-layout/MyShopLayout/component/ShopRegister/ShopRegister.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from "classnames/bind";
+import styles from "@/page-layout/MyShopLayout/component/ShopRegister/ShopRegister.module.scss";
+import Button from "@/common/components/Button/Button";
+
+import { useRouter } from "next/router";
+
+const cn = classNames.bind(styles);
+
+export default function ShopRegister() {
+  const router = useRouter();
+
+  const handleClick = () => {
+    return router.push(`/`); // 가게 정보 등록 페이지로 이동해야함. + 등록한 가게가 있으면 모달 띄우기
```

Comment:
return 키워드는 필요없지 않을까요?

Reply:
이렇게 되면 좋을 것 같습니다~
```suggestion
    return router.push('/'); // 가게 정보 등록 페이지로 이동해야함. + 등록한 가게가 있으면 모달 띄우기
```

Reply:
추가로 라우팅 경로는 상수화해서 다뤄주시면 좋습니다~

Reply:
아래와 같이 TODO주석으로 표현해주는게 좋을 것 같네요~
```suggestion
    // TODO: 가게 정보 등록 페이지로 이동해야함. + 등록한 가게가 있으면 모달 띄우기
    return router.push(`/`); 
```

Reply:
라우팅 경로 상수화는 이후에 페이지 이름을 정하면 진행할게요 !